cloud-sql/postgres/knex/connect-unix.js

```diff
@@ -0,0 +1,36 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+// [START cloud_sql_postgres_knex_connect_unix]
+const Knex = require('knex');
+
+const createUnixSocketPool = async config => {
+  const dbSocketPath = process.env.INSTANCE_UNIX_SOCKET;
```

Comment:
Let's add the secret manager disclaimer in these samples. See https://github.com/GoogleCloudPlatform/golang-samples/blob/main/cloudsql/postgres/database-sql/connect_tcp.go#L42-L45.